src/meta-schema/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const I18N_LOCALE_LANGUAGE = '_LOCALE_LANG';
```

Comment:
not sure about the naming... `locale` is a set of region-specific properties, it does not describe the source of a language information. Maybe `context`, `session`, `user`? In any case, `LANG` and `LANGUAGE` are confused in the docs.

Reply:
In my understanding, a language can also be part of these region-specific properties. The meaning here is language which is defined/derived by/from a locale.
Fixed it in docs already.

Reply:
ah ok, now that makes sense.